packages/text-field/index.tsx

```diff
@@ -278,10 +281,11 @@ class TextField<
     };
   }
 
-  inputProps(child: React.ReactElement<InputProps<T>>) {
+  get inputProps() {
     // ref does exist on React.ReactElement<InputProps<T>>
     // @ts-ignore
-    const {props} = child;
+    const {props} = React.Children.only(this.props.children);
```

Comment:
Why did you add this .only here? I don't think we should assume that it can only be one child. This will be a breaking change.

Reply:
```
renderInput() {
  const child: React.ReactElement<InputProps<T>> = React.Children.only(this.props.children);
  const props = this.inputProps(child);
  return React.cloneElement(child, props);
}
```
This is original code of renderInput function.
So I thought text-field was receiving only one intentionally.

Reply:
ah ok - if it was there before, then that is fine :)